image/templates/helm/stackrox-central/templates/_init.tpl.htpl

```diff
@@ -139,6 +139,14 @@
 {{ include "srox.autoSensePodSecurityPolicies" (list $) }}
 [<- end >]
 
+[< if .Operator >]
```

Comment:
Also on this section, do we want to handle "unknown" here as well or simply leave it empty within the metrics?

Reply:
This block as an `else`, which returns `manifest`.